com.rustdesk.RustDesk.json

```diff
@@ -0,0 +1,116 @@
+{
+  "id": "com.rustdesk.RustDesk",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "rustdesk",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=wayland",
+    "--socket=x11",
```

Comment:
You don't need both. See the note https://docs.flatpak.org/en/latest/sandbox-permissions.html#standard-permissions

Reply:
https://github.com/flathub/flathub/pull/5233/commits/6a93a2d10a10a152b4f58a0e49e9dbc51acc096a